spec/requests/current_user_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'CurrentUsers', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/current_user/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
Kindly remove this file or add it to your integrations folder and update it to pass. Leaving it inside the request folder will cause us errors in the future.